moveit_core/trajectory_processing/include/moveit/trajectory_processing/time_optimal_trajectory_generation.h

```diff
@@ -178,7 +178,7 @@ class TimeOptimalTrajectoryGeneration : public TimeParameterization
   /**
    * \brief Compute a trajectory with the given number of waypoints (plus or minus 1, due to rounding).
    * Resampling the trajectory to get the desired num_waypoints doesn't change the start and goal point,
-   * and all re-sampled waypoints will be on the path of the original trajectory (within the configured tolerances)
+   * and all re-sampled waypoints will be on the path of the original trajectory (within path_tolerance_)
```

Comment:
By following this chain:

https://github.com/ros-planning/moveit2/blob/f0d3202d637569ccac5fcc357a37e0d12a077c87/moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp#L1146

https://github.com/ros-planning/moveit2/blob/f0d3202d637569ccac5fcc357a37e0d12a077c87/moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp#L196

https://github.com/ros-planning/moveit2/blob/f0d3202d637569ccac5fcc357a37e0d12a077c87/moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp#L101

https://github.com/ros-planning/moveit2/blob/f0d3202d637569ccac5fcc357a37e0d12a077c87/moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp#L135

You can see the tolerance is a sum for all DoF. It would be good to work that into your comment somehow.

Reply:
I think it's basically the maximum distance from any desired waypoint, in meters. Which makes the default 0.1 meters very, very loose!

Reply:
Oh, I read the paper again. Everything is in configuration space, so the unit of `path_tolerance_` would usually be [rad].

https://smartech.gatech.edu/bitstream/handle/1853/44347/KunzRSS12-Trajectories.pdf?sequence=1&isAllowed=y

Reply:
Note, there are 3 versions of `computeTimeStamps()` that all should have this comment. I'll push a change and you can review it.